src/utilities/intl/tests/use-intl.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Simple test making sure our hook returns something. This test possibly may not be needed since hooks get tested with their implementation